ui/component/claimSupportButton/view.jsx

```diff
@@ -2,9 +2,7 @@
 import * as MODALS from 'constants/modal_types';
```

Comment:
- [ ] "Support" modal on own list incorrectly stuck at Fiat

Go to your own list and click "Support".  In production, it correctly shows "Boost" and LBC is selected, but in this branch, "Boost" is shown but only fiat is selectable.

<img width="555" alt="image" src="https://user-images.githubusercontent.com/64950861/173538285-894a1d4b-500d-4b37-9eea-a4fc6f98f421.png">